<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Обработка цифровых изображений</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color:rgb(255, 255, 255)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>798</width>
       <height>598</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#scrollAreaWidgetContents_4{
background-color:rgb(255, 255, 255)
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="lb_header">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Эквализация гистограммы и Линейное контрастирование</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="w_original" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="lb_Image1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_original">
             <property name="font">
              <font>
               <family>Segoe UI Black</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Оригинал</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lb_hist1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="w_modified1" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QLabel" name="lb_Image2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Эквализация гистограммы </string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lb_hist2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="w_modified2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="lb_Image3">
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="horizontalWidget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Линейное контрастирование</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lb_hist3">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="a_open"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Режим</string>
    </property>
    <addaction name="a_hist_and_contrast"/>
    <addaction name="a_hist_HSV_RGB"/>
    <addaction name="separator"/>
    <addaction name="a_add"/>
    <addaction name="a_multiply"/>
    <addaction name="a_power"/>
    <addaction name="a_logariphmic"/>
    <addaction name="a_negative"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="a_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="a_hist_and_contrast">
   <property name="text">
    <string>Эквализация и линейное контрастирование</string>
   </property>
  </action>
  <action name="a_hist_HSV_RGB">
   <property name="text">
    <string>Построение и эквализация гистограммы</string>
   </property>
  </action>
  <action name="a_add">
   <property name="text">
    <string>Добавить значение</string>
   </property>
  </action>
  <action name="a_multiply">
   <property name="text">
    <string>Умножить на значение</string>
   </property>
  </action>
  <action name="a_power">
   <property name="text">
    <string>Возведение в степень</string>
   </property>
  </action>
  <action name="a_logariphmic">
   <property name="text">
    <string>Логарифмическое преобразование</string>
   </property>
  </action>
  <action name="a_negative">
   <property name="text">
    <string>Негатив</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
